.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           pip install -r app/tests/requirements-tests.txt
 
       - name: Run Linters
-        if: ${{ false }}
```

Comment:
please remove the line completely since it is not needed

Reply:
Thought to leave it for simple enable-disable functionality.